pkgs/tools/system/hw-probe/default.nix

```diff
@@ -100,15 +100,16 @@ stdenv.mkDerivation rec {
         mesa-demos
         memtester
         sysstat # (iostat)
-        cpuid
         util-linuxMinimal # (rfkill)
         xinput
         libva-utils # (vainfo)
         inxi
         vulkan-utils
         i2c-tools
         opensc
-      ];
+      ]
+      # cpuid is only compatible with i686 and x86_64
+      ++ lib.optional (lib.elem stdenv.hostPlatform.system cpuid.meta.platforms) cpuid;
```

Comment:
Nice that it uses `cpuid.meta.platforms` rather than hardcoding `i686` and `x86_64` :+1: